game/scripts/vscripts/addon_game_mode.lua

```diff
@@ -333,6 +336,42 @@ function COverthrowGameMode:InitGameMode()
 	}
 end
 
+---------------------------------------------------------------------------
+-- Fix feed on tower
+---------------------------------------------------------------------------
+function COverthrowGameMode:OnEntityKilled(event)
+	local killer = EntIndexToHScript(event.entindex_attacker)
+	local death_unit = EntIndexToHScript(event.entindex_killed)
+	local unicKey = event.entindex_attacker + event.entindex_killed
+
+	if not _G.stopFeedOnTower[unicKey] then
```

Comment:
You probably want to check that `death_unit` is the main hero, otherwise it would trigger on any units (and illusions too) which can have their IDs reused. Something like that:

```suggestion
	if death_unit:IsRealHero() and PlayerResource:GetSelectedHeroEntity(death_unit:GetPlayerID()) == death_unit and not _G.stopFeedOnTower[unicKey] then
```

(same for damage filter)